src/port/port.tcl

```diff
@@ -2171,9 +2155,40 @@ proc action_info { action portlist opts } {
                 set label "$opt: "
             }
 
+            if {$ropt in {"description" "long_description"}} {
+                # These fields support newlines, we need to [join ...] to make
+                # them newlines
+                set inf [join $inf]
+            }
+
             # Format the data
             if { $ropt eq "maintainers" } {
-                set inf [unobscure_maintainers $inf]
+                set infresult {}
+                foreach serialized [macports::unobscure_maintainers $inf] {
+                    set parts {}
+                    array set maintainer $serialized
+
+                    if {[info exists maintainer(email)]} {
+                        lappend parts "Email: $maintainer(email)"
+                    }
+                    if {[info exists maintainer(github)]} {
+                        lappend parts "GitHub: $maintainer(github)"
+                    }
+                    if {[info exists maintainer(keyword)]} {
+                        switch $maintainer(keyword) {
+                            nomaintainer {
+                                lappend parts "no maintainer"
+                            }
+                            openmaintainer {
+                                lappend parts "open maintainer"
```

Comment:
I am not sure about these. Maybe I am just used to read them as keywords instead of multiple words… Also seems odd that they have no label.